### PR TITLE
Update IDE.md

### PR DESCRIPTION
**Pull request by songoo**
Creating new play 2.x projects in intelJ does not work, first there are deprecated errors in tests files, second IntelJ does not download 2.4 version but older one that is not working (no main class found). Only quick and straight solution is to create play 2.4 projects with Activator directly and then import. This should be mentioned in specification.

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @songoo,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: documentation/manual/gettingStarted/IDE.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ To create a Play application:
 
 IntelliJ IDEA creates an empty application using SBT.
 
+For play 2.4.x do not use IntelliJ wizard, but create your new project with Activator and then Import it to IntelliJ.
```
Comment (marcospereira) [inline documentation/manual/gettingStarted/IDE.md]: Maybe we can rephrase this to:

> Currently, for Play 2.4.x, instead of use the IntelliJ wizard to create a new project, we suggest that you create it using [Activator](https://www.typesafe.com/get-started) and then Import it to IntelliJ.

**Comment by typesafehub-validator** [comment]
Hi @songoo,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: documentation/manual/gettingStarted/IDE.md (hunk 1 above)
Comment (songoo) [inline documentation/manual/gettingStarted/IDE.md]: Hi, tnx for rephrase, i edited and commited.

Comment (songoo) [inline documentation/manual/gettingStarted/IDE.md]: Hmm, cant pass typesafehub-validator , when i click link , it wants name and email and shows github nickname, after submit it writes that its succesfully processed, but nothing hapends after that, tried it few times, but always with same result.

**Comment by jroper** [comment]
Not sure why the CLA validator isn't working, but I can confirm that @songoo has signed it.

File: documentation/manual/gettingStarted/IDE.md [hunk 2]
```diff
@@ -84,6 +84,8 @@ To create a Play application:
 
 IntelliJ IDEA creates an empty application using SBT.
 
+Currently, for Play 2.4.x, instead of use the IntelliJ wizard to create a new project, we suggest that you create it using Activator and then Import it to IntelliJ.
```
Comment (benmccann) [inline documentation/manual/gettingStarted/IDE.md]: use -> using

Comment (songoo) [inline documentation/manual/gettingStarted/IDE.md]: Changed in new commit. Typesafe is now finnaly working (green) , still fail on "The Travis CI build failed" - not sure what exactly it means.

**Comment by marcospereira** [comment]
Forward port here: c2b3380db1231941c18f59d41732867bd5350b72
